week2/week2/Program.cs

```diff
@@ -0,0 +1,256 @@
+﻿// See https://aka.ms/new-console-template for more information
+//Exercise 1
+using System.Diagnostics.Metrics;
+using System.Reflection.Metadata.Ecma335;
+
+Console.WriteLine("If & Switch");
+Console.WriteLine("Ex1");
+
+Console.WriteLine("Input first no:");
+int first = Convert.ToInt32(Console.ReadLine());
+Console.WriteLine("Input second no:");
+int second = Convert.ToInt32(Console.ReadLine());
+Console.WriteLine("Input third no:");
+int third = Convert.ToInt32(Console.ReadLine());
+
+if (first <= second)
+{
+    if (second <= third)
+    {
+        Console.WriteLine("Minimum is " + first);
+        Console.WriteLine("Maximum is " + third);
+    }
+    else if (first <= third)
+    {
+        Console.WriteLine("Minimum is " + first);
+
+        Console.WriteLine("Maximum is " + second);
+    }
+    else
+    {
+        Console.WriteLine("Minimum is " + third);
+        Console.WriteLine("Maximum is " + second);
+    }
+}
+
+else
+{
+    if (second >= third)
+    {
+        Console.WriteLine("Minimum is " + third);
+        Console.WriteLine("Maximum is " + first);
+    }
+    else if (first <= third)
+    {
+        Console.WriteLine("Minimum is " + second);
+        Console.WriteLine("maximum is " + third);
+    }
+    else
+    {
+        Console.WriteLine("Minimum is " + second);
+        Console.WriteLine("maximum is " + first);
+    }
+}
+
+//Exercise 2
+Console.WriteLine("Ex2");
+int max, min;
+max = (first > second && first > third) ?
+                first : (second > third) ? second : third;
+
+Console.WriteLine("Maximum is " + max);
+min = (first < second && first < third) ?
+                first : (second < third) ? second : third;
+Console.WriteLine("minimum is " + min);
+
+// Exercise 3
+Console.WriteLine("Ex3");
+Console.WriteLine("Input no of days: ");
+int no = Convert.ToInt32(Console.ReadLine());
+int years = no / 365;
+int months = (no % 365) / 30;
+int days = (no % 365) % 30;
+//if (years == 1)
+//{
+//    Console.WriteLine(years + "year"); }
+
+
+//if (months == 1)
+//{ 
+//    {
+//        Console.WriteLine(months + "month");
+//    }
+//    if (days == 1)
+//    {
+//        Console.WriteLine(days + "day");
+//    }
+//    else { Console.WriteLine(days + "days"); }
+
+//}
+//else { Console.WriteLine(months + "months"); }
+//else
+//{
+//    Console.WriteLine(years + "years");
+//    Console.WriteLine(months + "months");
+//    Console.WriteLine(days + "days");
+//} am incercat sa acopar cazurile in care rezultatul da an, luna, zi, dar m-am incurcat din nou in paranteze...
```

Comment:
It's great you've tried. 😄 Try to start small by doing simple exercises where the parentheses are easier to handle.